src/DIRAC/__init__.py

```diff
@@ -167,20 +169,105 @@ def _computeRootPath(rootPath):
 from DIRAC.ConfigurationSystem.Client.Config import gConfig
 
 
-def initialize():
-    """Load configuration.
+from DIRAC.Core.Security.Properties import SecurityProperty, UnevaluatedProperty
+from DIRAC.Core.Utilities import exceptions
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevel
+
+
+def initialize(
```

Comment:
@andresailer Currently this function isn't included in the code reference docs. I poked around in diracdoctools but it wasn't obvious to be how it could be included nicely without also affecting the documentation of extensions. Any suggestions?

Reply:
For completeness, to test I manually added `.. autofunction:: DIRAC.initialize` to `docs/source/CodeDocumentation/index.rst` which works.

Reply:
Do we want this to show up there https://dirac.readthedocs.io/en/latest/CodeDocumentation/index.html ? (It would feel a bit out of place)
Would there be a better place for it to be put?

Reply:
I agree that there should be a better place but I have no idea where 😅

Reply:
Core.Base.Client?

Reply:
Adding files to docs/diracdoctools/CustomizedDocs can be done, e.g.
```python
"""Additional docstring for the Client module."""
import textwrap
MODULE = "DIRAC.Core.Base.Client"
class CustomClient(object):  # pylint: disable=too-few-public-methods
    """Add the initialize function to Core.Base.Client docs"""
    def __init__(self):
        """Create a string containing restructured text documentation for all the special tranformation parameters
        :param str module: full path to the module
        :param bool replace: wether to replace the full docstring or not
           if you replace it completely add the automodule commands etc.!
        """
        self.module = MODULE
        self.replace = False
        self.doc_string = ""
        # Fill the docstring addition with what we want to add
        self.doc_string += textwrap.dedent(
            """
            .. autofunction:: DIRAC.initialize
            """
        )
CUSTOMIZED_DOCSTRINGS[MODULE] = CustomClient()  # pylint: disable=undefined-variable
```

Special treatment for `DIRAC/__init__.py`  would need more effort

Reply:
Nice! Done in f07467f95d0c9cf5010a0ce88dc7b6cbe902725f.

Reply:
Super!